### PR TITLE
[FEAT]: Implemented Timer component with breaks

### PR DESCRIPTION
**Pull request by shivanigangadharan**
Added timer component for pomodoro with short/long break, play/pause and stop feature.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,12 +23,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <title>React App</title>
```
Comment (mnath30) [inline public/index.html]: You can replace this title here with your app name

**Review by mnath30 — APPROVED** [review]
Code is well written and also got to learn about react-countdown from your code
Great work

File: public/index.html (hunk 1 above)
Comment (shivanigangadharan) [reply public/index.html]: Done

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 import './App.css';
 import Homepage from './pages/homepage';
+import TimerPage from './pages/timerPage';
 
 function App() {
   return (
-    <div>
-      <Homepage />
+    <div className="App">
+      {/* <Homepage /> */}
```
Comment (sharathnair9999) [inline src/App.js]: you can remove if this isn't needed

File: src/components/theTimer.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 100%;
+    padding-top: 5%;
+}
+.timer-section{
+    background: rgb(68, 68, 68);
+    width: max-content;
+    padding: 2% 5%;
```
Comment (sharathnair9999) [inline src/components/theTimer.css]: preferrably padding can be put in ems rems or px if <10px. % might be an issue in mobile devices

File: src/components/theTimer.js [hunk 4]
```diff
@@ -1,45 +1,81 @@
...
+                >
+                    {children}
+                </CountdownCircleTimer>
+                <div className="btn-div">
+                    {
+                        running ? <i className="fa-solid fa-pause icon pause" onClick={toggleClick}></i>
+                            : <i className="fa-solid fa-play icon" onClick={toggleClick}></i>
+                    }
+
+                    <i onClick={() => { setCurrentIndex(currentindex + 1); setRunning(false); Stop.play() }} className="fa-solid fa-square icon stop"></i>
```
Comment (sharathnair9999) [inline src/components/theTimer.js]: dont put `onClick` directly on `<i>` tag. instead wrap it in a `span` and put `onClick` in it

**Review by sharathnair9999 — APPROVED** [review]
Suggested trivial changes. Other than those everything looks to the point

File: src/components/theTimer.js (hunk 4 above)
Comment (shivanigangadharan) [reply src/components/theTimer.js]: done in [82c9913](https://github.com/shivanigangadharan/pomodoro/pull/1/commits/82c9913194c0673d22e6e4c5971eac1e643f763f)

File: src/components/theTimer.css (hunk 3 above)
Comment (shivanigangadharan) [reply src/components/theTimer.css]: done

File: src/App.js (hunk 2 above)
Comment (shivanigangadharan) [reply src/App.js]: done

**Pull request merged**